### ISSUE TITLE
How to download only the model(s) ?

### ISSUE THREAD

**Issue by javiabellan** [HyperInspire/InspireFace#106]
I want to download the model (or models) in a standard format (like pytorch, tensorflow, onnx, etc). There is any possibility of that?

Thanks.

**Comment by tunmx** [comment]
Hi, Some models, such as the original models (PyTorch or TensorFlow) for face detection and face recognition, can be obtained from [InsightFace](https://github.com/deepinsight/insightface) repository. Additionally, you can get the ONNX models for this project by using tar to extract them from the resource files in the project.

**Comment by javiabellan** [comment]
Thanks for answering, but i am interested in the face **atributes** like the ones appearing in this foto:

![](https://raw.githubusercontent.com/HyperInspire/InspireFace/refs/heads/master/images/banner.jpg)

More specifiacallly i am interested in the following attributes:
- **Left/Right eye**: open or close
- **Liveness**: Real or Fake
- **Blurry**: True or False *(this involves a Deep Learning model or is just an OpenCV-like algorithm ?)*
- **Age**: number or interval
- **Gender**: Male or Female
- **Race**: ["Black", "Asian", "Latino/Hispanic", "Middle Eastern", "White"]
- **Emotion**: "Happy", "Sad", "Angry",...
- **Mask**: True or False

How do you obtained the model that predict those attributes? Or where i can find this model(s) ?

> In the [attribute section of InsightFace](https://github.com/deepinsight/insightface/tree/master/attribute) only have a `gender_age`model.

**Comment by tunmx** [comment]
In fact, the models related to face attribute analysis is not yet open, and only the onnx model is currently provided

**Comment by tunmx** [comment]
These features are available in inspireface
